lib/jekyll/commands/doctor.rb

```diff
@@ -39,18 +39,21 @@ def healthy?(site)
             !conflicting_urls(site),
             !urls_only_differ_by_case(site),
             proper_site_url?(site),
-            properly_gathered_posts?(site),
+            properly_gathered_drafts_and_posts?(site),
           ].all?
         end
 
-        def properly_gathered_posts?(site)
+        def properly_gathered_drafts_and_posts?(site)
           return true if site.config["collections_dir"].empty?
-          posts_at_root = site.in_source_dir("_posts")
-          return true unless File.directory?(posts_at_root)
+          directories = ["_drafts", "_posts"]
+          directories.select do |directory|
+            File.directory?(site.in_source_dir(directory))
+          end
+          return true if directories.empty?
           Jekyll.logger.warn "Warning:",
-            "Detected '_posts' directory outside custom `collections_dir`!"
+            "Detected '#{directories.join("', '")}' directory outside custom `collections_dir`!"
           Jekyll.logger.warn "",
-            "Please move '#{posts_at_root}' into the custom directory at " \
+            "Please move '#{directories.join("', '")}' into the custom directory at " \
```

Comment:
I think it should be 

```
Please move '_drafts', '_posts' into the custom directory at..
```

Am I doing it wrong? I thought it would be better to combine this into single messages, rather than rendering the same message with different directories. What do you think, @ashmaroli?

Reply:
To discover (and learn Ruby along the way) the output to your code, you try simple code in your terminal itself..
  - In you terminal, run `irb`
  - Then execute `["_drafts", "_posts"].join("', '")` and see how the output looks like..

Reply:
Testing for `_posts` and `_drafts` separately is the best solution. For that create a new method for `_drafts` and then extract the common steps between the two into a `private` helper method..